docs/reference/sections/dom-binding.md

```diff
@@ -1,6 +1,21 @@
-Full documentation for the DOM Binding module is coming soon.
+DOM Binding is powered by Rivets.js, providing a flexible templating approach to cart rendering.
 
-In the meantime, you can check out the example on the [front page][] in combination with the [Rivets.js documentation][] to get an idea of how things work.
+See the [DOM Binding] section in the Guide for more.
 
-[front page]: /
-[Rivets.js documentation]: http://rivetsjs.com/
+### Rivets Binders
+
+A full breakdown of Rivets' available binders can be found under the Rivets docs [Binder Reference][].
```

Comment:
Ok - maybe a quick reference list of Rivets' binders, without going into detail of how to use each one? Given the close crossover with the docs rivets has, this would maybe be a watch-point to be aware of in the future if the version of Rivets included with Cart.js was updated, to make sure this was kept in sync with any new/updated binders.